apps/emqx_management/src/emqx_mgmt_api_data_backup.erl

```diff
@@ -261,15 +262,37 @@ data_export(post, #{body := Params}) ->
             {500, #{code => 'INTERNAL_ERROR', message => Msg}}
     end.
 
-data_import(post, #{body := #{<<"filename">> := FileName} = Body}) ->
+data_import(post, #{body := #{<<"filename">> := Filename} = Body}) ->
     case safe_parse_node(Body) of
         {error, Msg} ->
             {400, #{code => ?BAD_REQUEST, message => Msg}};
         FileNode ->
             CoreNode = core_node(FileNode),
-            response(
-                emqx_mgmt_data_backup_proto_v1:import_file(CoreNode, FileNode, FileName, infinity)
-            )
+            case
+                emqx_mgmt_data_backup_proto_v1:import_file(CoreNode, FileNode, Filename, infinity)
+            of
+                {ok, #{db_errors := DbErrs, config_errors := ConfErrs}} ->
+                    case DbErrs =:= #{} andalso ConfErrs =:= #{} of
+                        true ->
+                            {204};
+                        false ->
+                            DbErrs1 = emqx_mgmt_data_backup:format_db_errors(DbErrs),
+                            ConfErrs1 = emqx_mgmt_data_backup:format_conf_errors(ConfErrs),
+                            Msg = unicode:characters_to_binary(
+                                io_lib:format("~s", [DbErrs1 ++ ConfErrs1])
```

Comment:
no delemeter between the errors?

Reply:
I just moved what was already here...  :melting_face: :see_no_evil:

Reply:
apparently they are already formatted with new lines:

https://github.com/emqx/emqx/blob/4f3d70224f6f7e4742ff2e99a0feb5aee10d69be/apps/emqx_management/src/emqx_mgmt_data_backup.erl#L367-L369

https://github.com/emqx/emqx/blob/4f3d70224f6f7e4742ff2e99a0feb5aee10d69be/apps/emqx_management/src/emqx_mgmt_data_backup.erl#L997